src/handlers/gestures/eventReceiver.ts

```diff
@@ -36,6 +36,8 @@ const dummyStateManager: GestureStateManagerType = {
   },
 };
 
+const lastUpdateEvent: GestureUpdateEvent[] = [];
```

Comment:
Do we ever cleanup events stored for handler that are no longer in a tree? We perhaps should

Reply:
Changed in https://github.com/software-mansion/react-native-gesture-handler/pull/1807/commits/8ed3577f8640c5330fa52be049eef02aa4747b93. Events will be cleared after calling `onFinalize`.